src/OpenAL/OpenToolkit.OpenAL/OpenToolkit.OpenAL.csproj

```diff
@@ -9,6 +9,9 @@
   <PropertyGroup>
     <Version>4.0.0</Version>
     <Description>OpenAL bindings, and audio utility classes.</Description>
+    <ApplicationIcon />
+    <OutputType>Exe</OutputType>
```

Comment:
This looks bogus ;)

Reply:
It is, but it's only temp to get easy testing working

Reply:
Ah because of  `ALTest.cs`. Then this could just be pulled out into an actual Test project 8D

Reply:
This is fixed now. I split it into it's own project.